specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2022-12-01/arcSettings.json

```diff
@@ -0,0 +1,662 @@
+          },
+          {
+            "$ref": "#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ArcSettingNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ArcIdentityResponse"
+            }
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ArcSettingList": {
+      "description": "List of ArcSetting proxy resources for the HCI cluster.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of ArcSetting proxy resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ArcSetting"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to the next set of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ArcSettingsPatch": {
+      "description": "ArcSetting details to update.",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "description": "Resource tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "properties": {
+          "description": "ArcSettings properties.",
+          "$ref": "#/definitions/ArcSettingsPatchProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ArcSettingsPatchProperties": {
+      "description": "ArcSettings properties.",
+      "type": "object",
+      "properties": {
+        "connectivityProperties": {
+          "description": "contains connectivity related configuration for ARC resources",
+          "type": "object",
+          "items": {
+            "$ref": "#/definitions/ArcConnectivityProperties"
+          }
+        }
+      }
+    },
+    "ArcSetting": {
+      "description": "ArcSetting details.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "systemData": {
+          "description": "System data of ArcSetting resource",
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/systemData",
+          "x-ms-client-flatten": true,
+          "readOnly": true
+        },
```

Comment:
Since common-types/resource-management/v3/types.json has already contains systemData property in Resource definition and ProxyResource definition, and ArcSetting takes all properties from ProxyResource. We can remove the systemData definition here. 
Just a FYI. JS SDK breaking change is caused by x-ms-client-flatten, because Resource has a systemData property which doesn't have a x-ms-client-flatten, this flag here won't take effect now.  removie the definition here would cause less confusion.

Reply:
SystemData we have used in the last API version i.e. 2022-10-01 
Can we still keep it?

Reply:
Done